web/components/logos/Loading.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+export default function LoadingScreen() {
+
+    return (
+        <div className="flex flex-col bg-custom-background w-full h-screen justify-center items-center">
+            <div className="flex flex-row relative w-full h-[10%] justify-center mb-[10%] sm:w-[17%] sm:h-[13%] sm:mb-[5%]">
+                <img src = '/MBBLogo.png' className="shrink-0 stroke-[2.1px] object-contain sm:stroke-[3px]"></img>
+            </div>
+            <div className="flex flex-col w-full h-[10%] justify-center shrink-0">
+                <p className="text-white text-center font-opensans text-[30px] font-bold leading-normal tracking-[0.9px] uppercase">
```

Comment:
You can define properties related to the text in the parent `div` (or even a parent `p` if you use a `<br />`) so that they're not copied twice in case anything needs to change